main/webapp/modules/core/scripts/project/exporters.js

```diff
@@ -109,14 +109,6 @@ ExporterManager.stripNonFileChars = function(name) {
     return $.trim(name.replace(/[\\*\/:?"<>|]/g, ' ')).replace(/\s+/g, '-');
 };
 
-ExporterManager.handlers.exportTripleloader = function(format) {
-  if (!theProject.overlayModels.freebaseProtograph) {
-    alert($.i18n('triple-loader/warning-align'));
-  } else {
-    ExporterManager.handlers.exportRows(format, "txt");
-  }
-};
-
```

Comment:
`triple-loader/warning-align` is used only inside the definition of function exportTripleloader which is not used anywhere else.